src/styles.css

```diff
@@ -0,0 +1,45 @@
+body {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+}
+
+ul {
+    list-style-type: none;
+    column-count: 3;
+    max-width: 1000px;
+    border-radius: 5px;
+    border: 2px solid  lightgray;
+    background-color: #FAFAFA;
+    padding: 10px;
+}
+h3 {
+    display: inline-block;
+    padding: 10px;
+}
+
+.flag {
+    width: 50px;
+    display: inline-block;
+}
+
+.Africa {
```

Comment:
Volgens mij mis je hiervoor nog de functie in jouw app.js;

function regionColor(region) {
    if(region === "Africa") {
        return "blue"
    }
    if (region === "Americas") {
        return "green"
    }
    if (region === "Oceania") {
        return "purple"
    }
    if (region === "Europe") {
        return "yellow"
    }
    if (region === "Asia") {
        return "red"
    }
}